### PR TITLE
Bell icon changed as requested 

### PR DESCRIPTION
**Pull request by M4dhav**
Made requested changes as per #173

### CODE REVIEW

**Comment by M4dhav** [comment]
Swapped FontAwesomeIcons with Material3 because FontAwesomeIcons were not centered within the toggle circle

**Comment by guyluz11** [comment]
> Swapped FontAwesomeIcons with Material3 because FontAwesomeIcons were not centered within the toggle circle

It is not centered because  

We use `Icon()` istead of `FaIcon()` in toggle switch atom which should also get fixed

https://github.com/guyluz11/infinite_horizons/blob/9832cbb7d48139e28fa14d890d39df6ecf9e1cd8/lib/presentation/atoms/toggle_switch_atom.dart#L72


Can you change it as well?

**Comment by M4dhav** [comment]
Made the changes

File: lib/presentation/atoms/toggle_switch_atom.dart [hunk 1]
```diff
@@ -69,7 +70,7 @@ class _ToggleSwitchAtomState extends State<ToggleSwitchAtom> {
       values: const [false, true],
       onChanged: widget.disable ? null : onChange,
       iconList: [
-        Icon(
```
Comment (guyluz11) [inline lib/presentation/atoms/toggle_switch_atom.dart:72]: Good, you have another one in this page.
Please change it as well

**Review by guyluz11 — CHANGES_REQUESTED** [review]

**Review by guyluz11 — APPROVED** [review]

**Pull request merged**